examples/braket_features/Getting_Started_with_OpenQASM_on_Braket.ipynb

```diff
@@ -251,7 +251,7 @@
   },
```

Comment:
<p>Here, <code>requiresAllQubitsMeasurement</code> is <code>True</code>, which indicates that all qubits must be measured.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/540/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>